Cargo.lock

```diff
@@ -23,11 +23,11 @@ dependencies = [
  "parity-scale-codec",
  "serde",
  "sha3 0.8.2",
- "sp-core",
- "sp-io",
- "sp-runtime",
- "sp-runtime-interface",
- "sp-std",
+ "sp-core 3.0.0 (git+https://github.com/paritytech/substrate?branch=polkadot-v0.9.8)",
```

Comment:
what happened here, all the deps seem to have been changed like this

Reply:
I accidentally had the pallet depending on an older substrate version. It still compiled because it wasn't installed in any runtime yet.